### DISCUSSION TITLE
Sites Eding My Filers list

### DISCUSSION THREAD

**Discussion by MrKlatuu** [uBlockOrigin/uAssets#20974] [Q&A]
### Prerequisites

- [X] This is **NOT** a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL(s) where the issue occurs.

https://foxnews.com

### Description

It appears that certain websites can no edit the My Filters List.  I had a filter in there that I installed to block the "Sign in with Google" pop ups.  That has now been replaced with something completely different for Google, and I also noticed I am getting all these ads on the FOX News site - and there's a filter in My Filters for FOX as well!  How are they getting in to my PC and editing these personal filters?  For comparison, I have a brand New PC I am setting up, and the uBLock extension in THAT PC has nothing under My Filters, so using that as a comparison it's obvious these filters are getting added somehow.

### Other extensions used

I use Cookie Auto Delete, and Facebook Container along with uBLock Origin

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.54.0
Firefox: 115
filterset (summary):
 network: 103203
 cosmetic: 44006
 scriptlet: 19222
 html: 1126
listset (total-discarded, last-updated):
 default:
  user-filters: 1-0, never
  ublock-filters: 36744-227, 2m
  ublock-badware: 7527-10, 2m
  ublock-privacy: 925-4, 3m
  ublock-unbreak: 2202-1, 2m
  easylist: 75551-592, 3m
  easyprivacy: 33018-781, 3m
  plowe-0: 3783-1, 3m
  urlhaus-1: 9355-0, 3m
  ublock-quick-fixes: 278-63, 3m
filterset (user): [array of 1 redacted]
trustedset:
 added: [array of 5 redacted]
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 400 ms (selfie)
 maxAssetCacheWait: 197 ms
```
</details>

**Comment by BlazeFTL** [comment]
What You Are Saying Is Impossible...You Should Reset UBO To Default And Use Default List+Annoyance Only

**Comment by gorhill** [comment]
Websites cannot modify your settings in uBO.

Your comment is difficult to understand, just provide actual information: the content of _My filters_ pane, and the content of _Trusted sites_ pane.

**Reply by MrKlatuu** [reply]
Well - too late for that.  I erased all the My Filters, and put one back in that blocks the "Log In With Google" pop up.  I did NOT put the other three in there or remove the one I had.  So no idea how this happened, but it did! ;)  (Note that I have several PC's, and none of the others had this filter change in uBlock on them.  Go figure!)
This is the only "my Filers" I have installed:
||accounts.google.com/gsi/$3p
None of the offending sites are on the Trusted Sites list.

**Reply by gorhill** [reply]
> I did NOT put the other three in there or remove the one I had

As said, websites have no access to extension settings. If you want to believe otherwise, there is nothing more I can do here.
